src/components/Interpretations/common/UserMention/useUserSearchResults.js

```diff
@@ -0,0 +1,41 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import debounce from 'lodash/debounce'
+import { useEffect, useRef, useState } from 'react'
+
+const usersQuery = {
+    users: {
+        resource: 'users',
+        params: ({ searchText }) => ({
+            fields: 'id,displayName,userCredentials[username]',
+            query: searchText,
+            order: 'displayName:iasc',
+        }),
+    },
```

Comment:
This results in a paged request (good) with a default `pageSize` of `50`. I think we could probably lower this to about 10-20? Because it doesn't make sense to show such a long list.

However (regardless if we stick with 50 or go down to 20 results per page), a user can end up with a situation where `pager.total > pager.pageSize`. And if the user doesn't see the search result he/she was hoping for, he/she needs to refine the search term. Perhaps we should add a message for that too?

Reply:
@cooper-joe ?

Reply:
I don't think that a page size of `50` is necessarily a problem. I can see it being a good rescue in some edge cases, and it doesn't really degrade the normal, expected usage.

Agreed that the user does need to be alerted to `pager.total > pager.pageSize`. If we don't want to go down the route of adding infinite scrolling (not really worth it here, I think), then we could show a disabled style menu item as the last item with text `Too many results. Try refining the search.`

Reply:
Fixed.